### ISSUE TITLE
tf.lite.TFLiteConverter error

### ISSUE THREAD

**Issue by tsa10262** [tensorflow/tensorflow#28697]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 16.04
- TensorFlow version (use command below): tf-nightly 1.14.1.dev20190513
- Python version: 3.5.2

**Describe the current behavior**
I create a model from tensorflow, and it's a simple model.

The graph is as following:
1. Placeholder: type=tf.float32, shape=[None, 640, 480, 1]  (format: NHWC)
2. Conv2D: kernel_size = 5, filter_num = 32, stride=1, padding='same' ( tf.layer.con2d)
3. relu (tf.nn.relu)

I save this graph to test.pb. 

I want to convert this model to tflite model with int8, so i use tf.lite.converter to convert my test.pb to convert_test.tflite. 

But something strange...

When i use Netron to parse test.pb, the Conv2D layer is still Conv2D. But in my convert_test.tflite, the Conv2D layer is been changed to "DepthwiseConv2D".

What happen...?
And, what can i do?

**Comment by ANSHUMAN87** [comment]
@tsa10262 : That's expected behavior, based on your model ops, however it does not affect your output.
Please check for desired output, if it solves your purpose, then well and good. If my post helped resolve your issue, please close the thread, Thanks!

**Comment by tsa10262** [comment]
@ANSHUMAN87: Thanks for answering. It's not affect the output. 
I'll close the thread.

Thanks a lot.

**Comment by rankyung-hong** [comment]
@ANSHUMAN87 I have a follow-up question about this issue. I had a same issue. What should I do to get "Conv2D" layer instead of "DepthwiseConv2D"? 

First Conv2D layer is converted to "DepthwiseConv2D", but second Conv2D layer is converted to "Conv2D". I should have get every conv2d layers converted to conv2d due to post-processing steps.

model = keras.models.Sequential([
    keras.layers.Conv2D(filters=6, kernel_size=3, activation='relu', input_shape=(28, 28, 1), name="input_node"),
    keras.layers.MaxPooling2D(pool_size=(2, 2), strides=2),
    keras.layers.Conv2D(filters=16, kernel_size=3, activation='relu'),
    keras.layers.MaxPooling2D(pool_size=(2, 2), strides=2),
    keras.layers.Flatten(),
    keras.layers.Dense(120, activation='relu'),
    keras.layers.Dense(84, activation='relu'),
    keras.layers.Dense(10, activation='softmax', name='output_node')
])
